### PR TITLE
add new log message to main

### PR DESCRIPTION
**Pull request by floriangeyer21**

### CODE REVIEW

**Review by margarita-ignatieva — APPROVED** [review]

File: src/main/java/com/movie/store/Main.java [hunk 1]
```diff
@@ -63,8 +63,16 @@ public static void main(String[] args) throws AuthenticationException {
                 (UserService) injector.getInstance(UserService.class);
         AuthenticationService authenticationService =
                 (AuthenticationService) injector.getInstance(AuthenticationService.class);
-        log.info("Register new user " + authenticationService.register("best@email.ever", "1234"));
-        log.info("Find user by email " + authenticationService.login("best@email.ever", "1234"));
+        try {
+            authenticationService.register("best@email.ever", "1234");
+        } catch (Exception e) {
```
Comment (KseniiaMakarova) [inline src/main/java/com/movie/store/Main.java]: can your `register()` method throw an Exception?

File: src/main/java/com/movie/store/Main.java [hunk 2]
```diff
@@ -63,8 +63,16 @@ public static void main(String[] args) throws AuthenticationException {
...
+            log.error("Can't register new user. ", e);
+        }
+        try {
+            authenticationService.login("best@email.ever", "1234");
+        } catch (Exception e) {
+            log.error("Log in error. ", e);
```
Comment (KseniiaMakarova) [inline src/main/java/com/movie/store/Main.java]:
```suggestion
            log.warn("Log in error. ", e);
```
nothing really bad happened, `warn` level is enough

also, I can see one Sout left in this class below

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]
let's add logging to DaoImpl classes as well (when insert/update operations are successful)

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]
not all comments are fixed(

Я не увидел один комментарий сразу.

File: src/main/java/com/movie/store/dao/impl/ShoppingCartDaoImpl.java [hunk 3]
```diff
@@ -48,6 +49,8 @@ public ShoppingCart getByUser(User user) {
                             + "join fetch s.user "
                             + "where s.user.id = :id ", ShoppingCart.class);
             query.setParameter("id", user.getId());
+            log.info("Successfully update shopping cart entity with user id " + user.getId()
+                    + " and email " + user.getEmail());
```
Comment (KseniiaMakarova) [inline src/main/java/com/movie/store/dao/impl/ShoppingCartDaoImpl.java:53]: this is not the `update()` method

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**